projects/scion/toolkit/viewport/src/viewport.component.spec.ts

```diff
@@ -933,14 +936,14 @@ describe('Viewport', () => {
     component.setStyle('viewport-content', {
       'height': '600px',
     });
-    fixture.detectChanges();
+    await flushChanges(fixture);
 
     const viewportClientSizeCaptor = new ObserveCaptor<Dimension>();
     fromDimension$(component.viewport.viewportClientElement).subscribe(viewportClientSizeCaptor);
 
     expect(getSize(fixture, 'sci-viewport')).toEqual(jasmine.objectContaining({height: 300}));
     expect(getSize(fixture, 'div.viewport-client')).toEqual(jasmine.objectContaining({height: 600}));
-    expect(isScrollbarVisible(fixture, 'vertical')).toBeTrue();
+    expect(isScrollbarVisible(fixture, 'vertical')).toBe(true);
```

Comment:
Use `toBeTrue` since you don't provide an expectation fail output and to be consistent with the matcher below.

Reply:
done